src/wallet/rpcwallet.cpp

```diff
@@ -847,7 +847,10 @@ UniValue create_sidechain(const UniValue& params, bool fHelp)
             "                                          hexadecimal format. Used as public input for WCert proof verification. Its size must be " + strprintf("%d", SC_FIELD_SIZE) + " bytes\n"
             "}\n"
             "\nResult:\n"
-            "\"transactionid\"    (string) The resulting transaction id.\n"
+            "{\n"
```

Comment:
What about `sc_create` command output? We should change it similar to current one.
Do we need to keep both commands?

Reply:
Ok agreed.
For the time being I would keep both versions of the command.